js/index.js

```diff
@@ -1 +1,54 @@
 // Your code goes here
+
+const homeContainer = document.querySelector(".container home");
+homeContainer.addEventListener("click", (event)=>{
+ console.log("home container was clicked!");
+});
+
+const navLink = document.querySelectorAll(".nav-link");
+
+navLink.addEventListener("click", ()=>{
```

Comment:
navLink is a NodeList of the nav links you selected. On line 10 where you're implementing `addEventListener`, you're trying to add the event listener onto a list of iterable objects. You need to add the event listener to each object individually using a loop of some kind or `navLink[0].addEventListener` and so on.